### ISSUE TITLE
Suggest new functionalities that can be implemented

### ISSUE THREAD

**Issue by rahulkumaran** [rahulkumaran/Utlyz-CLI#30]
Please comment here to suggest new functionalities that you think should be implemented and give a one line reason as to why we should?

If it's convinving and the idea is actually good, then you'll get points and it'll be opened up as an issue

**Comment by Ace-Bansal** [comment]
A functionality can be added where the user can find out about any activity on the GitHub repo that they've starred or the repo that they're watching. This functionality will be useful specially during events like OpenCode.

**Comment by rahulkumaran** [comment]
What do you mean by any activity? Do you mean like getting updates when issues are added or PRs are made from people?

**Comment by Ace-Bansal** [comment]
Yes. So that the user can stay updated about that repo and find all the info in one place without having to open to repo each time.

**Comment by rahulkumaran** [comment]
I've actually thought of this but That's exactly what happens when you start watching a repo right? I like this idea, but can you improvise on it? Maybe think of how we could make it different?

**Comment by Ace-Bansal** [comment]
Sure, I'll get back to you if I come up with something :-)
